tests/integration/sharding_tests/test_sharding_backups.py

```diff
@@ -24,19 +23,43 @@
 SHARD_REL_NAME = "sharding"
 CONFIG_SERVER_REL_NAME = "config-server"
 S3_REL_NAME = "s3-credentials"
+SHARD_ONE_DB_NAME = "new-db"
+SHARD_ONE_COLL_NAME = "test_collection"
+SHARD_TWO_DB_NAME = "new-db-2"
+SHARD_TWO_COLL_NAME = "test_collection_2"
 TIMEOUT = 10 * 60
 
 
 @pytest.fixture()
-async def add_writes_to_db(ops_test: OpsTest):
-    """Adds writes to DB before test starts and clears writes at the end of the test."""
+async def add_writes_to_shards(ops_test: OpsTest):
+    """Adds writes to each shard before test starts and clears writes at the end of the test."""
     await writes_helpers.start_continous_writes(
         ops_test, 1, config_server_name=CONFIG_SERVER_APP_NAME
     )
     time.sleep(20)
     await writes_helpers.stop_continous_writes(ops_test, config_server_name=CONFIG_SERVER_APP_NAME)
+
+    # move continuous writes to shard-one
+    mongos_client = await generate_mongodb_client(
+        ops_test, app_name=CONFIG_SERVER_APP_NAME, mongos=True
+    )
+
+    mongos_client.admin.command("movePrimary", SHARD_ONE_DB_NAME, to=SHARD_ONE_APP_NAME)
+
+    # add writes to shard-two
+    write_data_to_mongodb(
+        mongos_client,
+        db_name=SHARD_TWO_DB_NAME,
+        coll_name=SHARD_TWO_COLL_NAME,
+        content={"horse-breed": "unicorn", "real": True},
+    )
+    mongos_client.admin.command("movePrimary", SHARD_TWO_DB_NAME, to=SHARD_TWO_APP_NAME)
+
     yield
     await writes_helpers.clear_db_writes(ops_test)
+    await writes_helpers.remove_db_writes(
```

Comment:
I believe these are exact duplicates, could we extend `clear_db_writes` with default arguments (`db`, `collection`) and make it more generic?